src/devtools/client/debugger/src/components/PrimaryPanes/Outline.js

```diff
@@ -86,26 +86,7 @@ export class Outline extends Component {
 
   setFocus(cursorPosition) {
     const { symbols } = this.props;
-    let classes = [];
-    let functions = [];
-
-    if (symbols && !symbols.loading) {
-      ({ classes, functions } = symbols);
-    }
-
-    // Find items that enclose the selected location
-    const enclosedItems = [...functions, ...classes].filter(
-      item => item.name != "anonymous" && containsPosition(item.location, cursorPosition)
-    );
-
-    if (enclosedItems.length == 0) {
-      return this.setState({ focusedItem: null });
-    }
-
-    // Find the closest item to the selected location to focus
-    const closestItem = enclosedItems.reduce((item, closest) =>
-      positionAfter(item.location, closest.location) ? item : closest
-    );
```

Comment:
Most of this part is doing the same thing as [`findClosestofSymbol`](https://github.com/RecordReplay/devtools/blob/master/src/devtools/client/debugger/src/utils/ast.js#L47). I think we can keep only one implementation.